system-parachains/constants/src/kusama.rs

```diff
@@ -133,3 +133,19 @@ pub mod fee {
 		<WeightToFee as frame_support::weights::WeightToFee>::weight_to_fee(weight)
 	}
 }
+
+pub mod snowbridge {
+	use frame_support::parameter_types;
+	use xcm::opaque::lts::NetworkId;
+
+	/// The pallet index of the Ethereum inbound queue pallet in the bridge hub runtime.
+	pub const INBOUND_QUEUE_PALLET_INDEX: u8 = 80;
```

Comment:
Added over here: https://github.com/polkadot-fellows/runtimes/pull/130/commits/c00fc1442d472949e0feeb1bf1c5f918f2b58d87

Reply:
IMO it is better to have it in one of the `bp-*` crates, because it is used in AH <> BH communications and those crates are exactly for that. But I'm not forcing you to change that - maybe it is fine to leave it here

Reply:
@claravanstaden I agree with Slava, I would move `INBOUND_QUEUE_PALLET_INDEX` to the `bp-bridge-hub-kusama` (the same for BHP) and also move that sanity [test](https://github.com/polkadot-fellows/runtimes/commit/c00fc1442d472949e0feeb1bf1c5f918f2b58d87#diff-bd26e40b81351dde58a2a2ef23c118cf04731f18bcda5ce2961b8df25026a5f3R649) directly to the bridge-hub-kusama-runtime, we did this exactly on other places, e.g. [here](https://github.com/polkadot-fellows/runtimes/blob/main/relay/kusama/src/lib.rs#L2796-L2798) or [here](https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/bridge-hubs/bridge-hub-kusama/src/bridge_to_polkadot_config.rs#L427-L432)

Reply:
@claravanstaden again agree with Slava ;)
This `constants` module, as I see it, is primarily dedicated to very common shared items among all parachains. The ETH<>K/P bridge constants are specific to a particular use case rather than being common. If anyone needs to use them, they can import the `bp-*` primitives crate.

Reply:
@svyatonik @bkontur I cleaned up the constants in https://github.com/polkadot-fellows/runtimes/pull/130/commits/0c6e184afa829c55e242623fb923defff61dcf2c. I did leave the tests in the integration tests instead of the runtime tests, because the constants are related to BH but needs to be compared to the AH runtime values. If the AH runtime constants are moved to `bp-asset-*` crates, I could move the tests to the runtime tests, but I didn't feel at liberty to start moving unrelated code around. Let me know your thoughts. :)

Reply:
@claravanstaden moved + small nits, please check/merge https://github.com/Snowfork/runtimes/pull/3